e2e/testdata/fn-render/fn-failure/Kptfile

```diff
@@ -0,0 +1,14 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: app
+pipeline:
```

Comment:
can you comment which function is meant to fail and which is mean to pass? What's the exit code of failing function? Does this short circuit?

There are separate test cases for testing structured and non-structured output which should test other aspects of how this failed.

Reply:
Renamed the starlark input file to `starlark-failure-fn.yaml` to make it obvious and also added a comment. Starlark function returns 1 exit code today for invalid starlark file.

By `short circuit`, do you mean bailing out immediately ? [Currently pipeline halts execution on the first function failure]

I plan to add testing of non-structured output in a separate PR where I am tweaking the output of returned when a `fn render` fails.